test/functional/job/job_spec.lua

```diff
@@ -101,9 +101,9 @@ describe('jobs', function()
     nvim('command', notify_job())
     nvim('command', "let j = jobstart('xxx', 'cat', ['-'])")
     local jobid = nvim('eval', 'j')
-    nvim('eval', 'jobsend(j, "abcdef")')
-    nvim('eval', 'jobstop(j)')
+    nvim('eval', 'jobsend(j, "abcdef\n")')
     eq({'notification', 'j', {{jobid, 'stdout', {'abcdef'}}}}, next_message())
+    nvim('eval', 'jobstop(j)')
```

Comment:
This defeats the purpose of the test  since now it is impossible for jobstop to cause 'exit' before 'stdout'.